cilium/cmd/service_update.go

```diff
@@ -50,7 +50,7 @@ func init() {
 	serviceUpdateCmd.Flags().BoolVarP(&k8sLoadBalancer, "k8s-load-balancer", "", false, "Set service as a k8s LoadBalancer")
 	serviceUpdateCmd.Flags().BoolVarP(&k8sHostPort, "k8s-host-port", "", false, "Set service as a k8s HostPort")
 	serviceUpdateCmd.Flags().BoolVarP(&localRedirect, "local-redirect", "", false, "Set service as Local Redirect")
-	serviceUpdateCmd.Flags().StringVarP(&k8sTrafficPolicy, "k8s-traffic-policy", "", "Cluster", "Set service with k8s externalTrafficPolicy as {Local,Cluster}")
+	serviceUpdateCmd.Flags().StringVarP(&k8sExtTrafficPolicy, "k8s-ext-traffic-policy", "", "Cluster", "Set service with k8s externalTrafficPolicy as {Local,Cluster}")
```

Comment:
Do we need to update/extend the following? https://github.com/cilium/cilium/blob/f59df8580b559aa122c3f0ff9d322360dbff7ef9/test/helpers/kubectl.go#L4165

Reply:
> and I wonder, if it's a test, why no tests have failed?

The e2e tests need an additional trigger. I just kicked them off. Think the error would have looked similar as [this one](https://github.com/cilium/cilium/issues/21392).

> BTW, do we want to keep --k8s-traffic-policy for backward compatibility?

Yeah, was wondering about the same. A quick search didn't find any external users/docs/publications etc (except our cmd reference), so I would question whether the flag has widespread usage. Let's rip it out, and add it back if anyone complains? :)